driver-core/build.gradle

```diff
@@ -33,13 +33,13 @@ sourceSets.main.resources.srcDirs = ['src/resources']
 dependencies {
     api project(path: ':bson', configuration: 'default')
 
-    api "com.github.jnr:jnr-unixsocket:$jnrUnixsocketVersion", optional
-    api "io.netty:netty-buffer:$nettyVersion", optional
-    api "io.netty:netty-transport:$nettyVersion", optional
-    api "io.netty:netty-handler:$nettyVersion", optional
-    api "org.xerial.snappy:snappy-java:$snappyVersion", optional
-    api "com.github.luben:zstd-jni:$zstdVersion", optional
-    api "org.mongodb:mongodb-crypt:$mongoCryptVersion", optional
+    implementation "com.github.jnr:jnr-unixsocket:$jnrUnixsocketVersion", optional
+    implementation "io.netty:netty-buffer:$nettyVersion", optional
```

Comment:
netty-buffer and netty-transport still need to be API because types from those are exposed in NettyStreamFactorFactory.  See the resolved comment just above.

Reply:
I overlooked the resolved comment. I have switched back to api for netty-buffer and netty-transport.